RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Row/NonFungibleAssetRow+View.swift

```diff
@@ -70,14 +71,38 @@ extension NonFungibleAssetList.Row.View {
 			}
 			.padding(.horizontal, .medium1)
 			.padding(.top, .large3)
-			.padding(.bottom, .medium1)
-			.background(.app.white)
+			.padding(.bottom, .medium1 + CGFloat(viewStore.stackedCards) * .small2)
 		}
 	}
 
 	private var headerHeight: CGFloat { HitTargetSize.small.frame.height + 2 * .medium1 }
 }
 
+private extension ViewStoreOf<NonFungibleAssetList.Row> {
+	var stackedCards: Int {
+		if self.isExpanded {
+			0
+		} else if self.resource.nonFungibleIdsCount == 1 {
+			1
+		} else {
+			2
+		}
+	}
+
+	var rowBackground: some View {
+		Group {
+			switch stackedCards {
+			case 0:
+				Color.white
+			case 1:
+				Image(.listBackgroundSimple).resizable()
+			default:
+				Image(.listBackgroundMultiple).resizable()
```

Comment:
This could just be a view containing two empty cards, rather than an image. But as I said, we should do it the proper way. Probably this list should be a UICollectionView anyway.

Reply:
Yes, the idea is to move this to a UIKit collection.

I tried making this view two empty cards, it didn't work as it wouldn't "exceed" the original card bounds.